l10n_ch_pain_base/views/bank_payment_line.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+
+    <record id="bank_payment_line_form" model="ir.ui.view">
+        <field name="name">l10n_ch_sepa.bank.payment.line.form</field>
+        <field name="model">bank.payment.line</field>
```

Comment:
This model has been removed from account_payment_order

https://github.com/OCA/bank-payment/commit/b8d818967bc221f1663bf8be99ffa4165f321414#diff-bc6e2817a362706bd1825d6d91ccc21ce12c159dd8ed779ff169c92bf3ad7512